documentation/install_from_source.markdown

```diff
@@ -8,6 +8,7 @@ layout: default
 [configure_heap]: ./configure.html#configuring-the-java-heap-size
 [module]: ./install_via_module.html
 [packages]: ./install_from_packages.html
+[running_tests]: /CONTRIBUTING.html#running-the-tests
```

Comment:
I would move it into the documentation folder and have a link to it in the README